src/main/java/frc/robot/subsystems/LlamaNeck2022.java

```diff
@@ -0,0 +1,93 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotConstants;
+import frc.robot.logging.RobotLogManager;
+import frc.robot.motors.MotorControllerEncoder;
+import frc.robot.motors.MotorControllerFactory;
+import frc.robot.motors.MotorType;
+import org.apache.logging.log4j.Logger;
+
+/** The llama neck subsystem, contains the llama neck motors and the limit switches. */
+public class LlamaNeck2022 extends SubsystemBase {
+  private static final Logger LOGGER = RobotLogManager.getMainLogger(LlamaNeck2022.class.getName());
+
+  private final MotorControllerEncoder llamaNeckMotor;
+  private final DigitalInput upperLimitSwitch;
+  private final DigitalInput lowerLimitSwitch;
+
+  /** The llama neck subsystem, contains the llama neck motors and the limit switches. */
+  public LlamaNeck2022() {
+    super();
+
+    llamaNeckMotor =
+        MotorControllerFactory.create(
+            RobotConstants.get().llamaNeck2022MotorID(), MotorType.SPARK_MAX_BRUSHLESS);
+    llamaNeckMotor.setInverted(RobotConstants.get().llamaNeck2022MotorInverted());
+    // upper switch, near the indexer wheel
+    upperLimitSwitch =
+        new DigitalInput(RobotConstants.get().llamaNeck2022UpperLimitSwitchChannel());
+    // lower switch, gives upper ball enough breathing room
+    lowerLimitSwitch =
+        new DigitalInput(RobotConstants.get().llamaNeck2022LowerLimitSwitchChannel());
+  }
+
+  /**
+   * Gets the status of the upper limit switch.
+   *
+   * @return true if pressed, false if not pressed
+   */
+  public boolean getUpperLimitSwitch() {
```

Comment:
I think the name `upperLimitSwitchIsPressed()` would be a clearer name. Similar for lower limit switch.